corehq/apps/export/static/export/js/list_exports.ng.js

```diff
@@ -80,6 +80,11 @@
             var currentUrl = useLegacyBulkExportUrl ? $scope.legacy_bulk_download_url : $scope.bulk_download_url;
             input.closest("form").attr("action", currentUrl);
         };
+        $scope.downloadRequested = function($event) {
+            var $btn = $($event.target);
+            $btn.addClass('disabled');
```

Comment:
Oh, it's a link, no `disabled` attribute, need to...switch the `href`, I suppose.